build.sbt

```diff
@@ -0,0 +1,307 @@
+import com.typesafe.tools.mima.plugin.MimaPlugin.mimaDefaultSettings
```

Comment:
yes, and the plugins are a pain to config without it. There is some autoImport thing that is now standard, so people assume you are using that.